CONTRIBUTING.md

```diff
@@ -46,12 +45,12 @@ must be given to the long term technical debt. Every new features should
 be covered by functional tests.
 
 When refactoring, structure your PR to make it easy to review and don't
-hesitant to split in multiple small, focused PRs.
+hestitate to split in multiple small, focused PRs.
```

Comment:
better: "hesitate to split it into multiple"

Reply:
fixed